### PR TITLE
1.4 Fixes

### PR DESCRIPTION
**Pull request by abnegate**
## What does this PR do?

- Add upgrade task
- Fix VCS/migrations/assistant scopes
- Fix domains migration
- Fix missing variable in email templates
- Fix call on missing route
- Fix missing mock headers endpoint
- Update console with scope fixes
- Update specs

## Test Plan

Test suite + manual tests

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by stnguyen90 — APPROVED** [review]

File: app/workers/mails.php [hunk 1]
```diff
@@ -35,9 +35,9 @@ public function run(): void
 
         $recipient = $this->args['recipient'];
         $subject = $this->args['subject'];
-        $name = $this->args['name'];
         $body = $this->args['body'];
         $variables = $this->args['variables'];
+        $name = $this->args['name'];
```
Comment (christyjacob4) [inline app/workers/mails.php:40]:
```suggestion
        $name = $variables['user'] ?? '';
```

File: app/controllers/api/migrations.php [hunk 2]
```diff
@@ -580,7 +580,7 @@
 App::get('/v1/migrations/firebase/connect')
     ->desc('Authorize with firebase')
     ->groups(['api', 'migrations'])
-    ->label('origin', '*')
+    ->label('scope', 'migrations.read')
```
Comment (christyjacob4) [inline app/controllers/api/migrations.php:583]: hmm shouldn't this be a write scope as there's an updateDocument happening here?

**Review by christyjacob4 — APPROVED** [review]

File: app/workers/mails.php (hunk 1 above)
Comment (abnegate) [reply app/workers/mails.php:40]: This should `args['name']` as it is used as `from` name, `variables['user']` is the `to` name

File: app/controllers/api/migrations.php (hunk 2 above)
Comment (abnegate) [reply app/controllers/api/migrations.php:583]: Makes senses, PR to update it [here](https://github.com/appwrite/appwrite/pull/6083)

**Review by christyjacob4 — APPROVED** [review]

**Pull request merged**